gem/lib/capybara-select2.rb

```diff
@@ -40,7 +40,7 @@ def select2(value, options = {})
       [value].flatten.each do |value|
         if find(:xpath, "//body").has_selector?("#{drop_container} li.select2-results__option")
```

Comment:
Thanks for the review.

Should it? Did you run into a situation where this was an issue? I am asking because the comment on line 42 says 'select2 version 4.0' which might indicate that the API is not always the same.